MapboxNavigation/RouteVoiceController.swift

```diff
@@ -110,10 +116,10 @@ open class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate {
         speechSynth.stopSpeaking(at: .immediate)
     }
     
-    func resumeNotifications() {
-        NotificationCenter.default.addObserver(self, selector: #selector(didPassSpokenInstructionPoint(notification:)), name: .routeControllerDidPassSpokenInstructionPoint, object: nil)
-        NotificationCenter.default.addObserver(self, selector: #selector(pauseSpeechAndPlayReroutingDing(notification:)), name: .routeControllerWillReroute, object: nil)
-        NotificationCenter.default.addObserver(self, selector: #selector(didReroute(notification:)), name: .routeControllerDidReroute, object: nil)
+    func resumeNotifications(by service: NavigationService) {
```

Comment:
Same suggestion as in https://github.com/mapbox/mapbox-navigation-ios/pull/2018/files#r265309688 - how about `observeNotifications(by:)` instead? Not a big deal, but I think it reads a bit more cleanly.